0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<p>It might be worth giving a little input about what <code>argnums=(0, 1)</code> is doing. (Telling JAX to calculate the partial gradients with respect to the first two parameters of <code>loss_function</code>, right?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='184'/>

Reply:
<p>Yep, added a description. Thnaks! </p>